assets/js/components/setup/setup-proxy.js

```diff
@@ -89,6 +89,28 @@ class SetupUsingProxy extends Component {
 			siteHostname,
 		} = this.state;
 		const isRevoked = 'revoked' === context;
+		const isSecondAdmin = isResettable;
+
+		let title;
+		let description;
+		let startSetupText;
+
+		if ( isRevoked ) {
+			title = sprintf(
+				/* translators: %s is the site's hostname. (e.g. example.com) */
+				__( 'You revoked access to Site Kit for %s', 'google-site-kit' ),
+				siteHostname
+			);
+			description = __( 'Site Kit will no longer have access to your account. If you’d like to reconnect Site Kit, click "Start Setup" below to generate new credentials.', 'google-site-kit' );
```

Comment:
Ah, now we'll need to set `startSetupText` here as well. Should probably be "Sign in with Google", since when you revoke the site is still "set up" overall.